src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Reissuance/ReissuanceConsent/HealthCertificateReissuanceConsentViewController.swift

```diff
@@ -13,9 +13,10 @@ class HealthCertificateReissuanceConsentViewController: DynamicTableViewControll
 		restServiceProvider: RestServiceProviding,
 		appConfigProvider: AppConfigurationProviding,
 		cclService: CCLServable,
-		certificates: [HealthCertificate],
+		certificates: [DCCCertificateContainerExtended],
 		healthCertifiedPerson: HealthCertifiedPerson,
 		didTapDataPrivacy: @escaping () -> Void,
+		didTapAccompanyingCertificate: @escaping ([HealthCertificate]) -> Void,
```

Comment:
```suggestion
		didTapAccompanyingCertificatesButton: @escaping ([HealthCertificate]) -> Void,
```
Otherwise I would expect that the user tapped on a single accompanying certificate